### PR TITLE
Fix help of some array functions

### PR DESCRIPTION
**Pull request by DelazJ**

### CODE REVIEW

File: resources/function_help/json/generate_series [hunk 1]
```diff
@@ -7,7 +7,7 @@
     {"arg":"stop", "description":"value that ends the sequence once reached"},
     {"arg":"step","optional":true,"default":"1","description":"value used as the increment between values"}
   ],
-  "examples": [ { "expression":"generate_series('1,5)", "returns":"array: '1', '2', '3', '4', '5'"},
-                { "expression":"generate_series('5,1,-1)", "returns":"array: '5', '4', '3', '2', '1'"}
+  "examples": [ { "expression":"generate_series(1,5)", "returns":"array: 1, 2, 3, 4, 5"},
+                { "expression":"generate_series(5,1,-1)", "returns":"array: 5, 4, 3, 2, 1"}
```
Comment (m-kuhn) [inline resources/function_help/json/generate_series]: Bad timing https://github.com/qgis/QGIS/pull/7692 ;)

Comment (m-kuhn) [reply resources/function_help/json/generate_series]: I also spotted and fixed them in my PR; I think if you stick to the first change only then they will both apply smoothly.

Comment (DelazJ) [reply resources/function_help/json/generate_series]: You sure? Your PR still shows `generate_series(**'**5,1,-1)`
Anyway, I'm not sure that there is a cleaner way to revert a commit on github web interface (i'm away from a git enabled machine) other than applying a new commit or opening new branch.

Comment (m-kuhn) [reply resources/function_help/json/generate_series]: I did something here:

https://github.com/qgis/QGIS/pull/7692/commits/ec4483a51a28763141b16c5fa7a3219de9972ef5#diff-b50741f513c3a914542b014ace3cc016

I'll merge it and we can see what happens ;)

**Pull request merged**